plugins/right-sizer/src/util/helpers.go

```diff
@@ -0,0 +1,11 @@
+package util
+
+// Contains returns true if the string is present in the slice.
+func Contains(list []string, wantedItem string) bool {
+	for _, item := range list {
+		if item == wantedItem {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
We like to use [go-funk](https://github.com/thoas/go-funk) for stuff like this

Reply:
I've made that change and removed my own `Contains` function.